Okta/OktaAuth/Tasks/OktaUserInfo.swift

```diff
@@ -20,7 +20,7 @@ internal class UserInfoTask: OktaAuthTask<[String:Any]> {
     }
     
     override func run(callback: @escaping ([String : Any]?, OktaError?) -> Void) {
-        guard let userInfoEndpoint = getUserInfoEndpoint(config) else {
+        guard let userInfoEndpoint = OktaEndpoint.userInfo.getURI(issuer: self.config.issuer) else {
```

Comment:
Why are we passing in `self.config.issuer` here, but not for our other endpoints?

Reply:
getUser API in OktaAppAuth looks now very alone :)
It's implementation depends on `OktaTokenManager` presence(access token). Probably we can just rename `OktaTokenManager` to `OktaAuthState`(or OktaAuthStateManager) and move this function to that class, so it will be in the right place
Thoughts?

Reply:
Agree. Do we need to approve this change first? Or Can I proceed with implementation?

Reply:
@jmelberg-okta , @nbarbettini , any objections on renaming `OktaTokenManager` -> `OktaAuthState`?

Reply:
@IldarAbdullin-okta I think I am good with it. I'd like to see an updated high-level class diagram of how these classes will interact though.

Reply:
@nbarbettini , thanks! I'm planning to start wiki this week

@anastasiiaiurok-okta , let's rename to `OktaAuthState` and move getUser `function` to that class

Reply:
Done.